jina/types/document/uid.py

```diff
@@ -37,6 +37,19 @@ def new_doc_hash(doc: 'DocumentProto') -> int:
     return id2hash(new_doc_id(doc))
 
 
+def get_document_hash(doc: 'DocumentProto') -> str:
+    """ Generate a new hexdigest based on the content of the document.
+
+    :param doc: a non-empty document
+    :return: the hexdigest based on :meth:`blake2b`
+    """
+    d = doc
+    if _doc_field_mask:
```

Comment:
does this _doc_field_mask exist?

Reply:
Is set as None by default. I'm not 100% how the doc_id was generated, but I'm taking this function exactly as the previous one `new_doc_id` that was doing the same, just a different name makes more sense since it's not setting the id anymore

Reply:
I can't find any reference of this `_doc_field_mask` being modified, only read.